### PR TITLE
Review of file

### PR DESCRIPTION
**Pull request by sabeena21**
### Change Description ###

Made minor modification to the content

### CODE REVIEW

**Comment by sabeena21** [comment]
@journey2w ,okay sure.
I have done only minor change that is changing RT IDE name to RTIDE just to maintain consistency.
Thank you

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Tizen Studio 2.4 Release Notes
+# Tizen Studio 2.4 Release Notes
```
Comment (journey2w) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: This line is duplicated.

**Comment by journey2w** [comment]
Please resolve conflict.

**Comment by sabeena21** [comment]
Closing this PR. With ref to PR #522

**Pull request closed without merging**